tests/baselines/reference/inlayHintsInferredTypePredicate1.baseline

```diff
@@ -0,0 +1,9 @@
+// === Inlay Hints ===
+function test(x: unknown) {
+                         ^
+{
+  "text": ": x is number",
+  "position": 25,
+  "kind": "Type",
```

Comment:
do I need to introduce a new `TypePredicate` kind for this?

Reply:
Nope: https://code.visualstudio.com/api/references/vscode-api#InlayHintKind